src/Components/SavedSearchAlert/Components/AddFiltersMedium.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react"
+import { Box, Checkbox, Flex, Spacer, Text } from "@artsy/palette"
+import { useSavedSearchAlertContext } from "Components/SavedSearchAlert/SavedSearchAlertContext"
+
+export const MEDIUM_OPTIONS = [
```

Comment:
Could we use the already existing medium options from [MediumFilter](https://github.com/artsy/force/blob/20165eac3335ff5e4a352d63cc4f7d394a51e210/src/Components/ArtworkFilter/ArtworkFilters/MediumFilter.tsx#L79) here?